packages/paste-website/src/assets/illustrations/HomeButtonIllustration.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 186 126"><g fill="none" fill-rule="evenodd"><path fill="#F4F4F6" d="M4 0h178a4 4 0 014 4v118a4 4 0 01-4 4H4a4 4 0 01-4-4V4a4 4 0 014-4z"/><rect width="111.798" height="34" x="37.596" y="46" fill="#0263E0" rx="4"/><path fill="#FFF" d="M55.404 60h76.181v7H55.404z"/></g></svg>
```

Comment:
We should name these in snake case format like https://github.com/twilio-labs/paste/pull/944/files#diff-468ed02f6cd79c9f905e682aaefb73054df57ee926d9f83d79639199c46ef676 since this naming scheme makes them clash with React components